pkg/netid/netid.go

```diff
@@ -0,0 +1,61 @@
+package netid
+
+/*
+Accessor methods to annotate namespace for multitenant support
+*/
+
+import (
+	"fmt"
+	"math"
+	"strconv"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+const (
+	// Current assigned VNID for the namespace
+	VNIDAnnotation string = "pod.network.openshift.io/multitenant.vnid"
+	// Desired VNID for the namespace
+	WantsVNIDAnnotation string = "pod.network.openshift.io/multitenant.wants-vnid"
```

Comment:
Are you proposing to add a new virtual resource that allows unprivileged admins to request VNIDS?  Because having these split doesn't add any security, and is basically inconsistent with our other admin commands.  Unless you have a proposal to have separate security models, we should just use `vnid` (that's consistent with every other allocator, like unix users, selinux labels, fsgroup, etc).